lib/chartmogul/csv/cancellation.rb

```diff
@@ -2,21 +2,17 @@
 
 module ChartMogul
   module CSV
-    # from https://chartmogul-samples.s3-eu-west-1.amazonaws.com/public/06_Cancellations.csv
-    CANCELLATION_FIELDS = %i[subscription_external_id customer_external_id date].freeze
-    CANCELLATION_HEADERS = %w[Subscription\ external\ ID Customer\ external\ ID Date].freeze
+    class Cancellation < Base
+      # from https://chartmogul-samples.s3-eu-west-1.amazonaws.com/public/06_Cancellations.csv
+      CANCELLATION_HEADERS = %w[Subscription\ external\ ID Customer\ external\ ID Date].freeze
 
-    Cancellation = Struct.new(*ChartMogul::CSV::CANCELLATION_FIELDS) do
-      include BaseStruct
-      extend CSVHeader
+      writeable_attr :subscription_external_id
+      writeable_attr :customer_external_id
+      writeable_attr :date
```

Comment:
@macharmi not that relevant right now, but how does it work when you want to submit multiple dates? Do you submit multiple lines? I guess you cannot just easily remove some date, right? We'll have to introduce some way to remove stuff later on, since we just always `upsert`.

Reply:
@pkopac we are not upserting for cancellation :sweat: 
```
    def import_options
      options = { validate: false }
      return options if @file_data_type.cancellation?

      options.merge(on_duplicate_key_update: { conflict_target: conflict_target, columns: :all })
    end
```

Reply:
@macharmi we need to be able to import multiple cancellation dates though :thinking:

Reply:
Multiple lines